test/README.md.test.tsx

```diff
@@ -2,14 +2,18 @@ import { promises as fs } from "fs";
 import { runTypeScriptFile } from "./utils/runTypeScriptFile";
 
 describe("README creation script", () => {
-  it("creates the README file of the repo", async () => {
+  afterAll(async () => {
+    await fs.unlink("./test/README.actual.md");
+  }, 20000);
```

Comment:
Do we need this? Having the file around in case the test fails was quite useful in the past. If need something like this, I would prefer to have it in a `beforeEach`

Reply:
Removed test cleanup